mix.exs

```diff
@@ -255,7 +256,8 @@ defmodule Livebook.MixProject do
       "docs/authentication/cloudflare.md",
       "docs/authentication/google_iap.md",
       "docs/authentication/tailscale.md",
-      "docs/authentication/custom_auth.md"
+      "docs/authentication/custom_auth.md",
+      "docs/how_to/nginx_https.md"
```

Comment:
I would put it under Deployment instead of introducing a new category, WDYT?

Reply:
I put it there because I already planned on having a "how-to guides" category.

I have a category taxonomy draft in my notes:

- Getting Started
- Tutorials
- How-to guides
- Features
- Core Concepts

I did this while thinking about how to organize the docs in the future. This is inspired by https://diataxis.fr/

I prefer keeping "how-to guides" inside their own category, independently of what they're about (I assume people would find them using the search feature, not browsing).

But I'd be ok with putting this one inside the "Deployment" category until we have more "how-to guides"

Reply:
The problem is that it feels inconsistent. If I want to learn how to deploy with Docker, I go to "Deployment". But for nginx, I go to "how-to"? I don't mind revisiting and migrating all guides to your proposed schema, but we should do it all once, instead of having some guides using the old and schema and some using the new one. :)